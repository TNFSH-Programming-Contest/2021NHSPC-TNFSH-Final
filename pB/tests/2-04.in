8 16 6
5 14 12 7 9 3
1 12 14 2 3 5
12 16 5 15 2 10
11 4 12 5 6 2
3 12 2 13 9 10
14 12 11 13 6 4
8 5 2 10 3 9
3 15 9 13 4 2
1 2
6 3 7 5 2 8 4
5 7 1 2 8 5
3 8 6 4
5 7 4 1 3 2
2 6 4
1 1
1 7
4 1 8 5 7
3 5 3 7
2 4 6
6 3 1 5 4 6 2
3 6 5 8
3 1 6 2
2 3 8
1 3
