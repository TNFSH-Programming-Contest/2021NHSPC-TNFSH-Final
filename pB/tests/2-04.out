1 2
0
2 5 8
0
1 1
0
0
1 7
0
0
1 4
1 3
0
1 6
0
0
